### PR TITLE
misc fixes and "improvements"

### PR DESCRIPTION
**Pull request by petaflot**
most are details that are pretty "obvious", some are arguable and are justified by the integration of https://github.com/petaflot/pygcode/blob/master/src/pygcode/tools/marlin_parse_MarlinDocumentation.py (does automatic parsing of the documentation to generate some classes) and should not cause any serious trouble.

accept the changes or don't, cherry pick it if you will... I've got enough on my hands not to argue about this. cheers!

### CODE REVIEW

File: _gcode/G000-G001.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ parameters:
 
 - tag: F
   optional: true
-  description: The maximum movement rate of the move between the start and end point. The feedrate set here applies to subsequent moves that omit this parameter.
+  description: The maximum movement rate (in mm/min) of the move between the start and end point. The feedrate set here applies to subsequent moves that omit this parameter.
```
Comment (thinkyhead) [inline _gcode/G000-G001.md:54]: If the current units are set to inches, then `F` indicates a feedrate in inches-per-minute.

File: _gcode/M906.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 tag: m0906
-title: Stepper Motor Current
+title: Trinamic Stepper Motor Current
```
Comment (thinkyhead) [inline _gcode/M906.md:3]: In older Marlin this pertains also to L64xx which are not made by Trinamic.

File: _gcode/M207.md [hunk 3]
```diff
@@ -47,3 +47,14 @@ parameters:
...
 Set lengths, feedrate, and Z lift for firmware-based retraction. See parameters below.
+
+### Retract/Recover tuning
+
+Typically one of these cases where the correct combination of 4 parameters is required for a good tuning. This is obviously tricky but JCZD wrote a test pattern to help with this while using a little filament as possible:
+
+![retract-recover helper](https://engrnage.ch/files/marlin_retract_recover_helper.png)
+
+* [3x3x3x3 pattern](https://engrnage.ch/files/retract_recover_3x3x3x3-jczd.gcode) - 81 individual samples, you probably want to start with this and it should give you a decent approximation
+* [4x4x4x4 pattern](https://engrnage.ch/files/retract_recover_3x3x3x3-jczd.gcode) - 256 individual samples, for fine-tuning
```
Comment (thinkyhead) [inline _gcode/M207.md:58]: Files and images need to be copied locally into the site. At the moment these links are broken.

**Comment by thinkyhead** [comment]
The domain that the files were linked to is gone. It is possible there was some kind of phishing going on. If we confirm that this is the case we'll get back to you. Thanks for the remaining changes, merging shortly!

**Comment by petaflot** [comment]
> The domain that the files were linked to is gone. It is possible there was some kind of phishing going on. If we confirm that this is the case we'll get back to you. Thanks for the remaining changes, merging shortly!

oops. it's supposed to be engrenage.ch (one 'e' missing). having some trouble merging/rebasing. might take me a few days, busy ATM

**Comment by petaflot** [comment]
@thinkyhead typo should be fixed

**Comment by thisiskeithb** [comment]
This PR has already been merged / closed, so you’ll need to create a new one with your fixes.

**Pull request merged**